src/components/Choropleth.vue

```diff
@@ -129,29 +138,46 @@ const style = (f: Feature) => {
 };
 
 const updateBounds = async () => {
-    // update bounds
+    const rawMap = toRaw(map.value);
+    if (!rawMap) return;
+    const leafletMap = rawMap.leafletObject;
+
     const country = selectedCountryId.value || "GLOBAL";
     const [west, east, south, north] = countryBoundingBoxes.value[country];
     const countryBounds = new LatLngBounds([south, west], [north, east]);
-    await map.value.leafletObject.fitBounds(countryBounds);
+    await leafletMap.fitBounds(countryBounds);
 
     // record bounds for testing
     bounds.value = { west, east, south, north };
 };
 
 const updateMap = async (newFeatures: Feature[]) => {
-    if (!map.value) return;
+    const rawMap = toRaw(map.value);
+    if (!rawMap) return;
+    const leafletMap = rawMap.leafletObject;
 
-    // remove layer from map
+    // remove layers from map
     geoJsonLayer.value.remove();
+    countryOutlineLayer.value?.remove();
 
     // create new geojson and add to map
     geoJsonLayer.value = geoJSON<FeatureProperties, Geometry>(newFeatures, {
         style,
         onEachFeature: configureGeojsonLayer
-    }).addTo(map.value.leafletObject);
+    }).addTo(leafletMap);
+
+    if (admin0Geojson.value) {
+        const latLngs = GeoJSON.coordsToLatLngs(admin0Geojson.value.geometry.coordinates, 2);
+        // className is just for testing
+        countryOutlineLayer.value = polyline(latLngs, {
+            color: "black",
+            weight: 1,
+            opacity: 0.5,
+            className: "country-outline"
+        }).addTo(leafletMap);
+    }
```

Comment:
Just a readability nitpick - could you either extract this to a function `addCountryOutline`, or add a comment just before?